src/locales/en-us/http/index.yaml

```diff
@@ -3,3 +3,6 @@ http:
     property:
       mtls: mTLS
       mTLS: mTLS
+common:
+  copyText: 'Copy'
+  copySuccessText: 'Copied!'
```

Comment:
Couple of things here:

1. If we are adding a new top-level property here then it shouldn't be in the `http`file.
2. Could we use more tree structures for these files, _preferably_ all lowercased? i.e.

```yaml
common:
  copy:
    idle: Copy
    success: Copied!
```

I'm curious as to why this shouldn't be in a component object rather than common? Where else would we use this?

Something resulting in feels like it would be better?

```yaml
component:
  copy-button:
    idle: Copy {text}
    success: Copied {text}!
```

Typically the individual component i18n files isn't super clear to me at the moment, but something similar to the above at least keeps component specific texts separate from shared texts.

Reply:
I’m following that it shouldn’t be in the http file, I’m going to change that, but why should this be in a component-scoped messages file?

How these strings are currently used, they very much act on a global level and aren’t specific to a specific module. This might change through time, but that’s not this PR.

Reply:
I suppose I don't understand what else apart from the `CopyButton` component would use these 2 pieces of text?

Something like `Save` for a button might end up in common (as the button component might use `Cancel`, or any number of things), but for something like a CopyButton, it pretty much always needs to say `Copy {text}` right? And I'm not aware of anything else that might need to use the same `Copy {text}` unless we already have another implementation of "copying text" somewhere?

Reply:
That’s not my point. It could well be that this is the only placing using these texts. My point is that there is not a more specific module which this would fit in.

It’s not specific to policies, data plane proxies, zones, services, etc. It’s a global or common component.

Reply:
Right for sure, I actually think we are on the same page here:

> It’s a global or common component.

Yeah exactly, its a common component called `copy-button`, its not specific to policies, data plane proxies, zones, services, hence `component.copy-button.idle`. I'm not totally clear on how we can pull in component specific i18n strings, but for the moment:


```vue
// CopyButton.vue
{{t('component.copy-button.idle')}}
```
is fine.

Having to do:

```vue
// SomethingElseView.vue
<CopyButton :text="t('common.copyText')" />
```
All the time when we know that the component is so specific that we will never want to change `Copy {text}` depending on where the component is used. Its always going to be `Copy {text}`.

This is one of the few instances where it makes sense to use `t` inside the component, because its a `CopyButton` that says `Copy`, its not a `DefinitionListItem` that doesn't ever say `Definition List`, or a `CopyButton` that sometimes says `Hello!`

Reply:
I can't really think of a place where `Copy {text}` / `t('component.copy-button.idle', {text: prop.name})` would need to be used outside of some sort of copying interaction, but 🤷 happy to leave it as is for the minute.